_posts/2019-05-30-bitcoin-sov-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+1. **Direct exchange**: Barter.
+
+2. **Indirect exchange**: When exchange is done in two steps using an intermediate good. Szabo's concept of "collectible" is a particular case within this category, which is using goods for long term indirect exchange.
+
+3. **Money**: When a good used for indirect exchange is widely accepted.
+
+We would deal separately with the concept of Unit of Account because this function fulfills the need for economic calculus which is an extremely important need, but different than fulfilling the need for exchange. Regarding the store of value function, it is not exclusive to money or proto-money, as all economic goods are by definition a better or worse store of value.
+
+So going back to Menger, we find that to overcome the problems of direct exchange (barter) humans began to use some goods for indirect exchange, that is, demanding or hoarding a good not for consumption but for exchanging it for other goods in the future. For example, if I have the possibility to decide on hoarding cereal or eggs, it is reasonable that I choose cereal as it is more durable and divisible. As the concept of medium of exchange is still not imaginable for me, I wouldn't even assess how cereal would be demanded by others to use it also for indirect exchange. I would just rely on others demanding it as food in the future (durable) and that I can use it for several different exchanges (divisible). **Once this experience is repeated, the concept of medium of exchange clearly reveals itself and it is immediately picked up by human entrepreneurship to be further exploited.**
+
+Each individual might choose completely different goods for indirect exchange depending on what they have available and when they are expecting to need them for exchange, so a convergence between individuals to choose the same good for this purpose is initially not necessary. The convergence would come naturally afterward, once humans have discovered the concept of medium of exchange, and that's when the most salable goods begin to compete for being a generalized medium of exchange (i.e. Money). The game theory concept of [focal point](https://en.wikipedia.org/wiki/Focal_point_(game_theory)) kicks in at this stage reinforcing the demand for the most salable good.
+
+A key concept in this debate is the term of the exchange. For a good to be money it is implicit that it serves well its purpose in the short term, which is also implicit in the words "most salable" within Menger's definition of money "the **most salable** of commodities", which [Carlos Bondone](http://www.carlosbondone.com/en/theory-of-economic-time/economic-theory/development-of-concepts.html) has generalized to "the most salable economic good". If it only serves its purpose in longer terms it might not be money but, it would still be a medium of indirect exchange.
```

Comment:
Add a period after "in the short term" and reword the following sentence to "This is also implicit in the words "most salable" within Menger's definition of money "the **most salable** of commodities", which [Carlos Bondone](http://www.carlosbondone.com/en/theory-of-economic-time/economic-theory/development-of-concepts.html) has generalized to "the most salable economic good".